collect_app/src/test/java/org/odk/collect/android/formentry/audit/FormSaveViewModelTest.java

```diff
@@ -68,38 +68,44 @@ public void setup() {
 
     @Test
     public void saveForm_returnsNewSaveResult_inSavingState() {
-        LiveData<FormSaveViewModel.SaveResult> saveResult1 = viewModel.saveForm(Uri.parse("file://form"), true, "", false);
-        assertThat(saveResult1.getValue().getState(), equalTo(SAVING));
+        viewModel.saveForm(Uri.parse("file://form"), true, "", false);
+        FormSaveViewModel.SaveResult saveResult1 = viewModel.getSavedResult().getValue();
+        assertThat(saveResult1.getState(), equalTo(SAVING));
 
-        LiveData<FormSaveViewModel.SaveResult> saveResult2 = viewModel.saveForm(Uri.parse("file://form"), true, "", false);
+        viewModel.saveForm(Uri.parse("file://form"), true, "", false);
+        FormSaveViewModel.SaveResult saveResult2 = viewModel.getSavedResult().getValue();
         assertThat(saveResult2, not(equalTo(saveResult1)));
     }
 
     @Test
     public void saveForm_wontRunMultipleSavesAtOnce() {
```

Comment:
I don't know if this makes sense anymore? Maybe it's best just to return when we're already saving rather than setting the result to `ALREADY_SAVING` just seems a little weird for the same `SaveResult` LiveData to go from the `SAVING` to `ALREADY_SAVING` state.

Reply:
Done.